cmake/tests.cmake

```diff
@@ -39,7 +38,7 @@ if(TARGET test_bitcoin)
       test_suite_name "${test_suite_macro}"
     )
     if(test_suite_name)
-      add_test(NAME ${test_suite_name}:${source_file}
+      add_test(NAME ${test_suite_name}
```

Comment:
Why is this going away?

Reply:
Because
```
  1/131 Test #102: txvalidationcache_tests:txvalidationcache_tests.cpp .........................   Passed    1.43 sec
```

looks too much verbose, doesn't it?

Reply:
But.. is that related to the param change here? Or is it just a random cosmetic change mixed in?

Reply:
> But.. is that related to the param change here?

Yes. We have no source file name-only for free. We have to extract it from an absolute path.